docker/py-dsci-100/Dockerfile

```diff
@@ -46,7 +42,20 @@ RUN jupyter labextension disable @jupyterlab/cell-toolbar-extension
 # Install the extension to remove the shift+M merge shortcut
 COPY shortcuts.jupyterlab-settings /home/${NB_USER}/.jupyter/lab/user-settings/\@jupyterlab/shortcuts-extension/shortcuts.jupyterlab-settings
 
-# Make sure everything in the home folder is owned by NB_USER
+# Copy jupyter_server_config.py which allows students to see and delete hidden files
+COPY jupyter_server_config.py /home/${NB_USER}/.jupyter
+
+# Copy gitconfig that sets global default pull strategy to rebase
+COPY .gitconfig /home/${NB_USER}/
```

Comment:
why?

Reply:
@briank-git please edit the commented out line on 48 above to reflect the default merge strat